packages/manager/modules/dbaas-logs/src/logs/detail/streams/subscriptions/streams-subscriptions.controller.js

```diff
@@ -0,0 +1,126 @@
+import datagridToIcebergFilter from '../../logs-iceberg.utils';
+
+export default class LogsStreamsSubscriptionsCtrl {
+  /* @ngInject */
+  constructor(
+    $stateParams,
+    $translate,
+    ouiDatagridService,
+    CucCloudMessage,
+    CucControllerHelper,
+    LogsHelperService,
+    LogsStreamsService,
+    LogsStreamsSubscriptionsService,
+  ) {
+    this.$stateParams = $stateParams;
+    this.$translate = $translate;
+    this.ouiDatagridService = ouiDatagridService;
+    this.CucCloudMessage = CucCloudMessage;
+    this.CucControllerHelper = CucControllerHelper;
+    this.LogsHelperService = LogsHelperService;
+    this.LogsStreamsService = LogsStreamsService;
+    this.LogsStreamsSubscriptionsService = LogsStreamsSubscriptionsService;
+
+    this.serviceName = this.$stateParams.serviceName;
+    this.streamId = this.$stateParams.streamId;
+    this.initLoaders();
```

Comment:
to move on `$onInit` section to be more consistency

Reply:
Done